app/views/events/show.html.haml

```diff
@@ -0,0 +1,22 @@
+.container-fluid
+  .col-lg-6
+    .panel
+      .panel-heading
+        .panel-body
+          %h1=@event.title
+          %p=@event.description
+          %p=link_to 'back', root_path
+  .col-lg-3
+    .panel
+      .panel-heading
+        .panel-body
+          %h4 Attendance list:
+          -@attendences.each do |event|
+            =event.user.email
+            =event.created_at
+          -if @attendences.where(user_id: current_user.id).count > 0
```

Comment:
this will cause an error when there's no user signed in, current_user would be nil. it should be modified as "current_user && @attendences.where(user_id: current_user.id).count > 0"